test/shmemx/Makefile.am

```diff
@@ -11,15 +11,9 @@
 # information, see the LICENSE file in the top level directory of the
 # distribution.
 
-check_PROGRAMS = \
-	shmemx_wait_until_all \
-	shmemx_wait_until_any \
-	shmemx_wait_until_some \
-	shmemx_test_all \
-	shmemx_test_any \
-	shmemx_test_some \
-	c11_test_shmemx_wait_until \
-	c11_test_shmemx_test
+check_PROGRAMS =
+
+if SHMEMX_TESTS
```

Comment:
We actually have three different batches of things that can be turned on independently here: profiling tests, portals4 tests, and shmem extensions tests.  This flag was already in the right place -- just around the shmemx tests.

Reply:
It's weird because some things in here are connected to the SOS build flags.

Reply:
It seems strange to me that turning off "SHMEMX_TESTS" should still result in building tests in the `test/shmemx` directory.  Is that really what we want?  No problem if so, I just find it a little weird.

Reply:
Effectively, `ENABLE_SHMEMX` is always enabled within SOS.  It's only for users of the external tests.

Reply:
Yeah, I noticed you have to change `configure.ac` directly to turn shmemx off.
Alright, cool - I'll push those changes shortly.